core/digital-twin-lib/src/utils/PermissionUtils.ts

```diff
@@ -346,8 +345,8 @@ export default class PermissionUtils {
     const containerConfigs = [];
     const bMailContent = await SharingUtils.getProfileShareBMail(vueInstance);
 
-    Object.keys(containerPermissions).forEach((containerAddress: string) => {
-      const shareConfigs = PermissionUtils.createShareConfig(
+    Object.keys(containerPermissions).forEach(async (containerAddress: string) => {
+      const shareConfigs = await PermissionUtils.createShareConfig(
```

Comment:
i just looked into it and I think there was a mistake converting "createShareConfig" into async function. Therefore is also no need to convert this foreach into async, which would probably not work if it was a "real" async function 😉